db/migrate/20181715075511_add_migration_status.rb

```diff
@@ -0,0 +1,8 @@
+class AddMigrationStatus < ActiveRecord::Migration
+  def change
+    add_column :minions, :tx_update_migration_available, :boolean, default: false, after: :tx_update_failed
+    add_column :minions, :tx_update_migration_notes, :string, after: :tx_update_migration_available
```

Comment:
Depending on the size of the notes we'd need to make this field bigger.

Reply:
it's just a url

Reply:
however if we would have a way to include the full release notes as text, we might be able to display the notes directly in velum, for air-gapped systems. but for now it's just a link... not sure how this will evolve

Reply:
By default `string` will map to `VARCHAR(255)`, I guess we should be fine?, we might want to tweak`limit` on the `add_column` call though so we are around something bigger (just in case the URL's grow at some point)